src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,64 @@
+.watch {
+  @include page-center;
+  @include watch-size;
+  border: 5px solid gray;
+  border-radius: 50%;
+
+  &__hand {
+    @include page-center;
+    animation-name: hands-movement;
+    animation-iteration-count: infinite;
+    animation-timing-function: steps(60, end);
```

Comment:
can we just combine them into a single property?

Reply:
I'm sure we can, but actualy I don't get how to scpecify animation-duration in this case? 

animation: hands-movement ```?``` steps(60, end) infinite;

So as, I use one common animation property for both of hands and separated class modifiers to apply duration...
Can you advise how it should be write down?